CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ should change the heading of the (upcoming) version to include a major version b
 
 # 5.17.1
 
+## @rjsf/utils
+
+- Added a new `skipEmptyDefault` option  in  `emptyObjectFields`, fixing [#3880](https://github.com/rjsf-team/react-jsonschema-form/issues/3880)
```

Comment:
Can you also move your two sections after `@rjsf/chakra-ui` to keep them in alphabetical order
```suggestion
- Added a new `skipEmptyDefault` option in `emptyObjectFields`, fixing [#3880](https://github.com/rjsf-team/react-jsonschema-form/issues/3880)
```